router/src/main/resources/Dockerfile

```diff
@@ -13,11 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # -----------------------------------------------------------------------
-
 FROM envoyproxy/envoy-alpine:v1.17.1
+FROM frolvlad/alpine-glibc:alpine-3.12_glibc-2.31
 
-COPY maven/envoy.yaml /etc/envoy/envoy.yaml
-COPY maven/security /home/wso2/security
+ENV LANG=C.UTF-8
+
+ARG MG_USER=wso2
+ARG MG_USER_ID=802
+ARG MG_USER_GROUP=wso2
+ARG MG_USER_GROUP_ID=802
+ARG MG_USER_HOME=/home/${MG_USER}
+
+ARG MOTD="\n\
+ Welcome to WSO2 Docker Resources \n\
+ --------------------------------- \n\
+ This Docker container comprises of a WSO2 product, running with its latest GA release \n\
+ which is under the Apache License, Version 2.0. \n\
+ Read more about Apache License, Version 2.0 here @ http://www.apache.org/licenses/LICENSE-2.0.\n"
+
+RUN \
+    addgroup -S -g ${MG_USER_GROUP_ID} ${MG_USER_GROUP} \
+    && adduser -S -u ${MG_USER_ID} -h ${MG_USER_HOME} -G ${MG_USER_GROUP} ${MG_USER} \
+    && mkdir ${MG_USER_HOME}/logs && chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/logs  \
```

Comment:
`chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/logs` We dont need this here at the moment. But I think we could keep it as it is and modify log_config.toml such that accesslogs are printed here.
https://github.com/wso2/product-microgateway/blob/3965bea3bfd2d5037d730ffc9ee6be836f596244/resources/conf/log_config.toml#L25